### ISSUE TITLE
Package tag staleness

### ISSUE THREAD

**Issue by jlixfeld** [UnusualAlpha/ib-gateway-docker#91]
**Describe the bug**
Stable is currently `10.22.2a` and Latest currently `10.23.2b` but tags are behind so the versions running in the images differ 

I see branches and PRs for the correct versions but sees those haven't made their way into images yet.

**To Reproduce**
```
% docker run --pull always -it ghcr.io/unusualalpha/ib-gateway:stable ls -al /root/Jts/ibgateway
stable: Pulling from unusualalpha/ib-gateway
Digest: sha256:7d5c3e5d0e49197f7abeab871d6c48e43996de8177d7cac2262cba0795367603
Status: Image is up to date for ghcr.io/unusualalpha/ib-gateway:stable
total 12
drwxr-xr-x 3 root root 4096 Jan 19  2023 .
drwxr-xr-x 3 root root 4096 Jan 19  2023 ..
drwxr-xr-x 5 root root 4096 Jan 19  2023 10.19.1j
%
```

**Comment by gnzsnz** [comment]
Hi @jlixfeld ,

I noticed the issue while working on a potential pull request.

I have created a copy of this repo and managed to automatically build new docker images for stable (10.19.2b) and latest (10.24.1b). you can find them [here](https://github.com/gnzsnz/ib-gateway-docker/pkgs/container/ib-gateway).

Please take into account that i build this to test my changes, so they are included in the new images. however i don't expect my changes to cause problems. I have:

- added support for IBC 3.18.0
- support for timezone
- support for ReloginAfterSecondFactorAuthenticationTimeout (IBC)
- support for AutoRestartTime (IBC)
- support for TWOFA_EXIT_INTERVAL (BIC) , this is currently hard coded in the image

so basically these images are on par with IBC 3.18.0 plus timezone support.

I will submit a pull request to merge this changes.

**Comment by jlixfeld** [comment]
Closing — no longer tracking this on my end. Feel free to reopen if it's still relevant.
